GooglePlayInstant/LoadingScreen/LoadingScreenScript.cs

```diff
@@ -31,15 +35,20 @@ private IEnumerator Start()
 
             if (loadingScreenConfigJsonTextAsset == null)
             {
-                throw new FileNotFoundException(
-                    "Error on Loading Screen: Could not find LoadingScreenConfig.json file in Resources folder.");
+                throw new FileNotFoundException("LoadingScreenConfig.json missing in Resources folder.");
             }
 
-            var loadingScreenConfigJson = loadingScreenConfigJsonTextAsset.ToString();
+            var loadingScreenConfig = loadingScreenConfigJsonTextAsset.ToString();
 
-            var loadingScreenConfigJsonObj = JsonUtility.FromJson<LoadingScreenConfig>(loadingScreenConfigJson);
+            var loadingScreenConfigJsonObj = JsonUtility.FromJson<LoadingScreenConfig>(loadingScreenConfig);
```

Comment:
I think if you look back at my previous comment here I was hoping for this variable to become loadingScreenConfig (the variable name would match the type name). Please change line 41 back to loadingScreenConfigJson.

Reply:
Oh, I was a little confused to be honest. I should have asked. But yeah resolved.